CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v2.2.0](https://github.com/voxpupuli/puppet-network/tree/v2.2.0) (2024-01-22)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-network/compare/v2.1.0...v2.2.0)
+
+**Implemented enhancements:**
+
+- Support puppet-filemapper 4.0.0 [\#319](https://github.com/voxpupuli/puppet-network/pull/319) ([silug](https://github.com/silug))
+
+**Closed issues:**
+
+- When will puppet-network support filemapper 4.0.0? [\#318](https://github.com/voxpupuli/puppet-network/issues/318)
```

Comment:
I added the skip-changelog label to the issue nevauae it doesn't add any value to the changelog. Please regenerate.

Reply:
Done.